src/components/Users/User/User.tsx

```diff
@@ -0,0 +1,96 @@
+import React, {useState} from 'react';
+import Button from 'components/core/Button/Button';
+import Input from 'components/core/Input/Input';
+import UserModel from 'model/UserModel';
+import classes from './User.module.scss';
+
+interface UserProps {
+  user : UserModel;
+  editUser: (id: string | number, user: UserModel) => void;
+  deleteUser: (id: string | number) => void;
+}
+
+const User: React.FC<UserProps> = ({user, editUser, deleteUser}) => {
+
+  const [toggleDetails, setToggleDetails] = useState<boolean>(false);
+  const [userValues, setUserValues] = useState<UserModel>(user);
+
+  const changeUserValueHandler = (value : string,  prop?: string) =>{
+    setUserValues(prevUserValues => ({...prevUserValues, [prop!]: value}));
+  }
+
+
+  return (
+    <div className={classes['c-user']}>
+      <div className={classes['c-user__short-info']} onClick={()=>setToggleDetails(!toggleDetails)}>
+        <div>{userValues.id}</div>
+        <div>{userValues.email}</div>
+        <div>{userValues.phoneNumber}</div>
+      </div>
+      {toggleDetails ? 
+            <div className={classes['c-user__all-details']}>
+            <div className={classes['c-user__input-holder']}>
+              <Input
+                type={'text'}
+                placeholder={'first name'}
```

Comment:
Could you please format this placeholder to be `First name` or `Please enter first name`